DevelopGitHub/GitHubExtension/GitHubExtension.Tools.LocalizationTool/Translate/Language.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+
+namespace GitHubExtension.LocalizationTool.Translate
+{
+    public enum Language
+    {
+        Uk,
+        En,
+        Ru
+    }
+
+    public static class LanguageExtension
+    {
+        public static Language GetLanguageEnum(this string language)
```

Comment:
I remembered you asked not to use extension method for string in same cases. I'll change it.